chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertificationBodyController.java

```diff
@@ -53,14 +52,14 @@
 
     @ApiOperation(value = "List all certification bodies (ACBs).",
             notes = "Setting the 'editable' parameter to true will return all ACBs that the logged in user has "
-                    + "edit permissions on. The logged in user must have ROLE_ADMIN or ROLE_ONC to see retired ACBs."
-                    + "The default behavior of this service is to list all of the ACBs in the system that are not retired.")
+                    + "edit permissions on. Security Restrictions:  Everyone can see all active.  ROLE_ADMIN and "
+                    + "ROLE_ONC_ADMIN can see deleted.")
```

Comment:
I don't think anyone can see deleted any more

Reply:
I think the previous documentation was correct except maybe that it did not specifically call out anonymous users.

Reply:
I was simply cutting and pasting the security restrictions from the spreadsheet.  The code is correct and the  Swagger annotations should be correct now.